src/components/common/Button/index.tsx

```diff
@@ -0,0 +1,17 @@
+import * as S from "./Button.styled";
+import { ButtonStyleProps } from "./type";
+
+interface ButtonProps extends ButtonStyleProps {
+  onClick: () => void;
+  children: React.ReactNode;
+}
+
+const Button = ({ variant = "primary", size = "full", onClick, children }: ButtonProps) => {
+  return (
+    <S.Button styles={{ variant, size }} onClick={onClick} disabled={variant === "disabled"}>
```

Comment:
styles는 인라인 스타일 객체와 혼동이 있을 것 같아서 일일이 넣어주는 방식이 좋을 것 같아요.
```suggestion
    <S.Button variant={variant} size={size} onClick={onClick} disabled={variant === "disabled"}>
```

Reply:
[4566981](https://github.com/woowacourse/react-shopping-cart/pull/361/commits/4566981793598645721739cd970c77497ef6b196)

수정완료했습니다!